guides/source/5_0_release_notes.md

```diff
@@ -120,6 +120,9 @@ Please refer to the [Changelog][railties] for detailed changes.
     ([Pull Request](https://github.com/rails/rails/pull/22457),
      [Pull Request](https://github.com/rails/rails/pull/22288))
 
+*   Introduced an `after_bundle` callback for use in Rails plugin templates.
```

Comment:
We can remove this from here. It is not a kind of feature that I'd give highlight.